### ISSUE TITLE
System.Diagnostics.Tests.ProcessTests.CheckUserAndGroupIds test failure

### ISSUE THREAD

**Issue by jkotas** [dotnet/corefx#35708]
Failed in https://github.com/dotnet/corefx/pull/35606/checks?check_run_id=71345269 on Ubuntu.1604.Amd64.Open-x64-Release:

```
Child exception:
  Xunit.Sdk.EqualException: Assert.Equal() Failure
Expected: HashSet<UInt32> [4, 20, 24, 25, 27, ...]
Actual:   HashSet<UInt32> [4, 20, 24, 25, 27, ...]
   at System.Diagnostics.Tests.ProcessTests.CheckUserAndGroupIds(String userId, String groupId, String groupIdsJoined, String checkGroupsExact) in /__w/1/s/src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs:line 515

Child process:
  System.Diagnostics.Process.Tests, Version=4.2.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 System.Diagnostics.Tests.ProcessTests Int32 CheckUserAndGroupIds(System.String, System.String, System.String, System.String)

Child arguments:
  1000, 1000, 4,20,24,25,27,29,30,44,46,109,110,998,1000, True
```

**Comment by stephentoub** [comment]
cc: @tmds

**Comment by danmosemsft** [comment]
Xunit could be more useful than 
```
Expected: HashSet<UInt32> [4, 20, 24, 25, 27, ...]
Actual:   HashSet<UInt32> [4, 20, 24, 25, 27, ...]
```

**Comment by jkotas** [comment]
Failed again in https://github.com/dotnet/corefx/pull/35720

**Comment by danmosemsft** [comment]
Doesn't seem to be failing today.

**Comment by wfurt** [comment]
note the:
```
Child arguments:
  1000, 1000, 4,20,24,25,27,29,30,44,46,109,110,998,1000, True
```

This may be fixed by #35680

**Comment by stephentoub** [comment]
Closing as dup of https://github.com/dotnet/corefx/issues/35933
